PhoneBook.Domain/Location.cs

```diff
@@ -30,12 +29,12 @@ public class Location : BindableDomainObject
 
     public static Location NewObject()
     {
-      return DomainObject.NewObject<Location>();
+      return NewObject<Location> ();
```

Comment:
By convention, we typically prefix static members with the type where they are declared.